pkg/informer/informer.go

```diff
@@ -30,6 +39,28 @@ type GenericInformer struct {
 	retries       int64             // Counts times we have tried without establishing a watch.
 }
 
+type Resource struct {
+	ApiGroups []string `yaml:"apiGroups"`
+	Resources []string `yaml:"resources"`
+}
+
+func GetAllowDenyData(cm *v1.ConfigMap) ([]Resource, []Resource) {
+
+	var allow []Resource
+	allowerr := yaml.Unmarshal([]byte(cm.Data["AllowedResources"]), &allow)
+	if allowerr != nil {
+		klog.Errorf("Unmarshal: %v", allowerr)
```

Comment:
Let's be more descriptive with these 2 error messages to help the end user debug the problem.
```
klog.Errorf("Error while parsing allowed resources from ConfigMap. Can't use configured value, defaulting to allow all resources. %v", allowerr)
```

Reply:
Good point!